### PR TITLE
Add ability to set target on copyright link

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Add ability to set target on copyright link so that it can be set to open in a new tab

### How to review 
Check that target gets set on the link correctly

### CODE REVIEW

**Comment by rmccar** [comment]
[Add functionality for target attribute to be set on the copyright declaration](https://trello.com/c/1nk4XbRh/773-add-functionality-for-target-attribute-to-be-set-on-the-copyright-declaration)

File: src/components/footer/_macro.njk [hunk 1]
```diff
@@ -221,7 +221,7 @@
                 {% if params.copyrightDeclaration is defined and params.copyrightDeclaration %}
                 <div class="grid">
                     <div class="grid__col u-mt-m">
-                        <p class="u-fs-s u-mb-no">&copy; {{ params.copyrightDeclaration.copyright }} <br> {{ params.copyrightDeclaration.text }} <a href="{{ params.copyrightDeclaration.url }}">{{ params.copyrightDeclaration.link }}</a>.</p>
+                        <p class="u-fs-s u-mb-no">&copy; {{ params.copyrightDeclaration.copyright }} <br> {{ params.copyrightDeclaration.text }} <a href="{{ params.copyrightDeclaration.url }}" target="{{ params.copyrightDeclaration.target }}">{{ params.copyrightDeclaration.link }}</a>.</p>
```
Comment (jrbarnes9) [inline src/components/footer/_macro.njk:224]: Does this need an `if` statement in case target is not defined?

Comment (jrbarnes9) [reply src/components/footer/_macro.njk:224]: Will need `rel="noopener"` if `target` is defined, so could bundle both attributes with `params.newWindow` like we do in button component...
```
if params.newWindow is defined and params.newWindow %}target="_blank" rel="noopener"{% endif %}
```

Comment (rmccar) [reply src/components/footer/_macro.njk:224]: yep ill sort that out

**Review by boxadesign — APPROVED** [review]

**Pull request merged**